src/index.ts

```diff
@@ -5,21 +5,19 @@ import { fetchLastIssueInfo } from './fetch-last-issue-info';
 if (require.main === module) {
   fetchLastIssueInfo({
     inputs: {
+      githubRepository: utils.getInput('repository', { required: true }),
+      githubToken: utils.getInput('token', { required: true }),
```

Comment:
```suggestion
      githubToken: utils.getInput('token', { required: process.env.NODE_ENV !== 'development' }),
```

Do you want that for `repository` as well? I'm not sure how this is meant to work without the action providing inputs, but `labels`/`state` is working however you're testing it, so perhaps you meant you don't need the token for local testing? (_which I guess could be a hassle?_)

Reply:
`repository` is mandatory. We just need to change that flag on development for `token`

Reply:
> We just need to change that flag on development for `token` because we don't need that for public repos

Verified.

The current action runs fine in public repo without providing the token, but the `GITHUB_TOKEN` ENV itself does not exist. I guess there is something else that affects the request in private repos requiring a token.